### PR TITLE
misc fixes, including da.around and da.isclose

### PR DESCRIPTION
**Pull request by shoyer**

### CODE REVIEW

File: dask/array/core.py [hunk 1]
```diff
@@ -597,7 +597,15 @@ def store(self, target, **kwargs):
...
-    __float__ = __int__ = __bool__ = __complex__ = compute
+    def __int__(self):
+        return int(self.compute())
+    def __bool__(self):
+        return bool(self.compute())
+    __nonzero__ = __bool__  # python 2
+    def __float__(self):
+        return float(self.compute())
+    def __complex__(self):
+        return complex(self.compute())
```
Comment (mrocklin) [inline dask/array/core.py]: This is good.  Thanks.

File: dask/array/core.py [hunk 2]
```diff
@@ -1220,6 +1228,11 @@ def modf(x):
 modf.__doc__ = np.modf
 
 
+@wraps(np.around)
+def around(x, decimals=0):
+    return map_blocks(x, partial(np.around, decimals=decimals), dtype=x.dtype)
```
Comment (mrocklin) [inline dask/array/core.py]: Consider `elemwise(np.around, x, 0, dtype=x.dtype)`

Comment (mrocklin) [inline dask/array/core.py]: Or rather, now that I look below and see that you know about elemwise, why didn't elemwise work here?

Comment (shoyer) [inline dask/array/core.py]: Just didn't seem like there was any need to use elemwise and invoke the overhead of broadcasting, etc, given that around is a unary operation -- note that the decimals argument needs to be an int, not an array.

**Pull request merged**